cmd/virt-launcher-monitor/virt-launcher-monitor.go

```diff
@@ -417,10 +417,13 @@ func removeArg(args []string, arg string) []string {
 
 func dumpLogFile(filePath string) {
 	f, err := os.Open(filePath)
-	if err != nil {
+	if err != nil && errors.Is(err, os.ErrNotExist) {
+		return
+	} else if err != nil {
```

Comment:
Is this related to a shift of passt into libvirt?

Reply:
Not directly. passt log can be printed to the journal or to a file (libvirt is not forwarding passt stderr). Since we don't have journal in virt-launcher we choose the log file option.
Before moving passt to libvirt we could read at least the startup logs from the passt stderr, but any other logs after the initial startup weren't there (since after startup passt is detached). Therefore, the change is related to the shift to libvirt but not directly,

Reply:
@EdDev done